app/modules/individuals/resources.py

```diff
@@ -677,3 +680,27 @@ def post(self, task_id):
             [target_individual], request_data, NotificationType.individual_merge_complete
         )
         return {'vote': vote, 'merge_completed': True}
+
+
+@api.route('/merge_conflict_check')
+class IndividualMergeConflictCheck(Resource):
+    def post(self):
+        from flask_login import current_user
+
+        if not current_user or current_user.is_anonymous:
+            abort(code=401)
+        if not isinstance(request.json, list):
+            abort(message='must be passed a list of individual ids', code=500)
```

Comment:
These generate no frontendFault audit logs unlike the HoustonException. Not entirely sure if they would be useful

Reply:
yup, i chose to treat these very lightly, as this api is very nearly just a GET with no affect on the system (i made it a POST only cuz there is the "potential" for a long list of IDs being queried).  hence recorded nothing in audit.  probably could go either way.